build_tools/bazel/iree.bazelrc

```diff
@@ -152,6 +152,7 @@ build:generic_clang --copt=-Wunused-comparison
 build:generic_clang --copt=-Wvla
 
 # Treat backrefs during linking as errors.
+build:generic_clang --linkopt=-fuse-ld=lld
```

Comment:
I think this is supposed to be controlled since it is not supported in the Apple environment. See IREE_ENABLE_LLD.

Reply:
Oh, it's not meant to be included this PR. Will remove it.

Reply:
Removed